src/System.Management.Automation/engine/SpecialVariables.cs

```diff
@@ -321,25 +326,29 @@ internal static class SpecialVariables
                                                                    /* PSCommandPath */     typeof(string),
                                                                  };
 
-        internal static readonly string[] PreferenceVariables = {
-                                                                    SpecialVariables.DebugPreference,
-                                                                    SpecialVariables.VerbosePreference,
-                                                                    SpecialVariables.ErrorActionPreference,
-                                                                    SpecialVariables.WhatIfPreference,
-                                                                    SpecialVariables.WarningPreference,
-                                                                    SpecialVariables.InformationPreference,
-                                                                    SpecialVariables.ConfirmPreference,
-                                                                };
-
-        internal static readonly Type[] PreferenceVariableTypes = {
-                                                                    /* DebugPreference */       typeof(ActionPreference),
-                                                                    /* VerbosePreference */     typeof(ActionPreference),
-                                                                    /* ErrorPreference */       typeof(ActionPreference),
-                                                                    /* WhatIfPreference */      typeof(SwitchParameter),
-                                                                    /* WarningPreference */     typeof(ActionPreference),
-                                                                    /* InformationPreference */ typeof(ActionPreference),
-                                                                    /* ConfirmPreference */     typeof(ConfirmImpact),
-                                                                  };
+        internal static readonly string[] PreferenceVariables =
+        {
+            SpecialVariables.DebugPreference,
+            SpecialVariables.VerbosePreference,
+            SpecialVariables.ErrorActionPreference,
+            SpecialVariables.WhatIfPreference,
+            SpecialVariables.WarningPreference,
+            SpecialVariables.InformationPreference,
+            SpecialVariables.ConfirmPreference,
+            SpecialVariables.PSNativeCommandUseErrorActionPreference,
```

Comment:
I remember we have some open issues for the variables. Do we really want to add the new one in the list?

Reply:
See [here](https://github.com/PowerShell/PowerShell-RFC/blob/6fb05cdcf7b1a9808844acd26dcfb7a0c458faa1/1-Draft/RFC00XX-Native-Command-Error-Handling.md#psnativecommanduseerroractionpreference)

Reply:
I don't understand the reference to RFC. My intention is to draw attention to a possible problem.

Reply:
The RFC prescribes the creation of a new preference variable and this PR implements that. The discussion around whether to create a preference variable and what to call it has already been concluded.